### DISCUSSION TITLE
For issues with

### DISCUSSION THREAD

**Discussion by mykeln** [GAM-team/got-your-back#425] [General]
Lots of docs/instructions out there for troubleshooting the admin_policy_enforced issue, but none worked for me.

It wasn't until I went to https://admin.google.com/ac/owl/settings and changed the setting to: Internal apps
Trust internal apps that everything worked perfectly!

**Comment by TheBigForrest** [comment]
I am getting the following error:
Access blocked: GYB has not completed the Google verification process


GYB has not completed the Google verification process. The app is currently being tested, and can only be accessed by developer-approved testers. If you think you should have access, contact the [developer](https://accounts.google.com/).
If you are a developer of GYB, see [error details](https://accounts.google.com/).
Error 403: access_denied

Your help is appreciated
